client/src/data_layer_configs/population_by_census_block_config.ts

```diff
@@ -72,38 +122,32 @@ export const styleLayer: FillLayer = {
 
 // TODO(kailamjeter): finalize content https://app.shortcut.com/blueconduit/story/5682/cleanup-fe.
 const popupInfo: PopupInfo = {
-  title: 'Census block',
-  subtitle: 'ACS Census block data',
+  title: 'U.S. demographic information',
+  subtitle: 'Data derived from U.S. Census data',
   detailsTitle: 'Demographic information',
   featureProperties: [
     {
-      label: 'Geographic identifier (GEOID)',
-      name: 'census_geo_id',
-      dataType: FeaturePropertyDataType.String,
-    },
-    { label: 'Block name', name: 'census_block_name', dataType: FeaturePropertyDataType.String },
-    {
-      label: 'Total population',
+      label: 'Total residents',
       name: 'total_population',
       dataType: FeaturePropertyDataType.Number,
     },
     {
-      label: 'Population under the age of 5',
+      label: 'Residents under the age of 5',
       name: 'under_five_population',
       dataType: FeaturePropertyDataType.Number,
```

Comment:
should these be percentages? as in can we convert them in the FE to %.

Reply:
Added a todo! Definitely agree that's easier to read